RussianFilter/sections/specific.txt

```diff
@@ -32,6 +32,7 @@ news.allcrimea.net###kinotvukrnet
 tut.by##div[data-io-title="Новости компаний"]
 masterpaiki.ru###secondary .widget_execphp
 masterpaiki.ru##a[href^="http://epnclick.ru/redirect"]
+filmitorrent.site##a[href^="https://tiruniversal.com"]
```

Comment:
better block by URL or by more universal selector
`filmitorrent.site##.post > center > a[target="_blank"] > img`

Reply:
@Alex-302,
changed the rule.